manifests/02-deployment.yaml

```diff
@@ -95,3 +98,9 @@ spec:
           items:
             - key: ca-bundle.crt
               path: tls-ca-bundle.pem
+      - name: bound-sa-token
```

Comment:
Could you add a comment / TODO explaining that this service account token is inert for now (as you mentioned in your commit message)?

Reply:
done with 9fec421 , added a comment on the usage of the definition of the volume.

Reply:
Looks like bindata needs to be updated and committed from the changes to `9fec421`